Governance/tech/conventions.md

```diff
@@ -1,147 +1,295 @@
+* Every data variable should include for each of its dimensions and each of its scalar coordinate variables the cell_methods information of interest (unless this information would not be meaningful). 
+* It is especially recommended that cell_methods be explicitly specified for each spatio-temporal dimension and each spatio-temporal scalar coordinate variable.
 ```
 
-```{dropdown} missing and valid data related 
-CF -
-_FillValue, missing_value, valid_min, valid_max, and valid_range attributes to indicate missing data. Missing data is allowed in data variables and auxiliary coordinate variables. Generic applications should treat the data as missing where any auxiliary coordinate variables have missing values; special-purpose applications might be able to make use of the data. Missing data is not allowed in coordinate variables.
+```{dropdown} **missing and valid data** 
+**CF - recommended**<br>
+[`_FillValue`, `missing_value`, `valid_min`, `valid_max`, and `valid_range`](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#missing-data) attributes are used to indicate missing data. Missing data is allowed in data variables and auxiliary coordinate variables. Generic applications should treat the data as missing where any auxiliary coordinate variables have missing values; special-purpose applications might be able to make use of the data.<br>
+Missing data is not allowed in coordinate variables.<br>
+
+**-FillValue**<br>
+The [`_FillValue`](https://docs.unidata.ucar.edu/netcdf-c/current/attribute_conventions.html#autotoc_md89) attribute specifies the fill value used to pre-fill disk space allocated to the variable, it is then returned when reading values which were never written. 
+If `_FillValue` is defined then it should be scalar and of the same type as the variable. If the variable is packed using scale_factor and add_offset attributes, the _FillValue attribute should have the data type of the packed data.<br>
+If not defined the deafult fill value for the type of the variable is used. However, use of the default fill value for data type byte is not recommended.<br>
+Note that changing the value of this attibute, won't changed previously 'filled' data automatically.<br>
+If `valid_range` is specified `_FillValue` should be outside of this range.
+
+**missing-value**<br>
+The `missing_value` is not treated in any special way by the netCDF library, but it may be used by specific applications. The missing_value attribute can be a scalar or vector containing values indicating missing data. These values should all be outside the valid range.<br>
+When scale_factor and add_offset are used, the value(s) of the missing_value attribute should be specified in relation to the packed data, so that missing values can be detected before the scale_factor and add_offset are applied.<br>
+If both missing_value and _FillValue are used, they should have the same value.
```

Comment:
Why is that? I actually assumed they could be different, e.g. for precip data over land to differentiate between 'never known here' (land mask _FillValue) and 'not measured at this timestep and location' (over land missing-value).

Reply:
Are you asking about the last point? "If both missing_value and _FillValue are used, they should have the same value"
I used the conventions document as a reference and often just their same wording. I'll open an issue as it might be easier to merge and then work this out, there must be some masking example. By the way that's exactly the kind of feedback I was hoping for :-)

Reply:
haha you're welcome ;-)
Yes I don't know, like it would make sense, one less thing for rendering tools to deal with (seeing both should be treated as `NaN`), I'm just surprised that there isn't (recommended) a way to differentiate between masking and unmeasured.